python/src/data_processing/setup_database.py

```
@@ -0,0 +1,137 @@
+import pyodbc
+
+# Define connection details
+server = 'Chaitanya'  # Use your server name from Step 1
+database = 'synthea_db'  # Use your database name from Step 3
+driver = 'ODBC Driver 17 for SQL Server'  # Using ODBC Driver 17
+
+# Connection string using Windows Authentication
+conn_str = f"DRIVER={driver};SERVER={server};DATABASE={database};Trusted_Connection=yes;"
+
+try:
```

Just let it blow up.  You don't need a try catch here.  You are also swallowing all the errors so no matter what happens it will be reported as a connection error.